packages/actor-function-factory-not/lib/Not.ts

```diff
@@ -0,0 +1,17 @@
+import { TermSparqlFunction } from '@comunica/bus-function-factory';
+
+import {
+  bool,
+  declare,
+  SparqlOperator,
+} from '@comunica/expression-evaluator';
+
+export class Not extends TermSparqlFunction {
+  protected arity = 1;
```

Comment:
I don't think so, it does remind me that these should be readonly. The fields are just there to describe the declared function better. 
(And they are used by the abstract class to generate proper errors)

Reply:
> it does remind me that these should be readonly

This might be an indication that this approach is not strict enough.

I was thinking something like this:
```
const function = new TermSparqlFunction({
  arity: 1,
  operator: SparqlOperator.NOT,
  overloads: declare(SparqlOperator.NOT)
    .onTerm1(() => val => bool(!val.coerceEBV()))
    .collect()
});
```

Reply:
Ah yes, might be better than using an abstract class